xdsl/dialects/builtin.py

```diff
@@ -300,6 +308,13 @@ def string_value(self):
             root += "." + ref.data
         return root
 
+    def print_builtin(self, printer: Printer):
+        printer.print_string("@")
+        printer.print_identifier_or_string_literal(self.root_reference.data)
```

Comment:
This kind of thing is what I was talking about for the `print_symbol` helper, feels like a useful thing to reuse to me.